resources/js/alpine/linkifyDirective/linkifyDirective.ts

```diff
@@ -0,0 +1,20 @@
+import linkifyHtml from 'linkify-html';
+import type { DirectiveData, DirectiveUtilities } from 'alpinejs';
+
+export function linkifyDirective(
+  el: HTMLElement,
+  _: DirectiveData,
+  { effect }: DirectiveUtilities,
+) {
+  effect(() => {
+    el.innerHTML = linkifyHtml(el.innerHTML, {
+      formatHref: {
+        url: (href) => `/redirect?url=${href}`,
```

Comment:
We should probably also add the redirect intermediary page to `[url=]` shortcode tags (and linkified raw URLs in shortcode).

Reply:
Updated `[url=]`, `[link]` and auto-url embeds in latest.